crypto/ec/ecp_nistp256.c

```diff
@@ -2176,10 +2168,9 @@ int ec_GFp_nistp256_points_mul(const EC_GROUP *group, EC_POINT *r,
                 ECerr(EC_F_EC_GFP_NISTP256_POINTS_MUL, ERR_R_BN_LIB);
                 goto err;
             }
-            num_bytes = BN_bn2bin(tmp_scalar, tmp);
+            num_bytes = BN_bn2lebinpad(tmp_scalar, g_secret, sizeof(g_secret));
         } else
```

Comment:
Nit: {} around else branch too

Reply:
Done, I added {} around else branches matching if with {} also in the other instances in this file and p521, p224: 8efb3ebc4001f13d79f547e996a612375d9ec08d

Reply:
LGTM, my approval still holds.